src/modules/stakewise/stakewise-daemon/tasks/send-exit-data.go

```diff
@@ -88,7 +95,17 @@ func (t *SendExitData) Run() error {
 			t.logger.Warn("Error getting private key", slog.String(PubkeyKey, pubkey.HexWithPrefix()), log.Err(err))
 			continue
 		}
+		if key == nil {
+			t.logger.Warn("Private key is nil", slog.String(PubkeyKey, pubkey.HexWithPrefix()))
```

Comment:
You can probably make this more user friendly with a message like "Private key not found - you may need to regenerate it f you're sure you own it."